From 2d0a42abc2bb21773d6487607f387a9c271b2718 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Fri, 25 Jul 2025 10:57:38 +0200
Subject: [PATCH] dt-bindings: gpio: fix misspelling of "STMicroelectronics"

"STMicroelectronics" company name is misspelled in the title of STMPE GPIO
bindings. Fix it.

Change-Id: I8201b434b83fc0c701d0114a51d1ec60fc6dc585
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/477729
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 Documentation/devicetree/bindings/gpio/st,stmpe-gpio.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/gpio/st,stmpe-gpio.yaml
+++ b/Documentation/devicetree/bindings/gpio/st,stmpe-gpio.yaml
@@ -4,7 +4,7 @@
 $id: http://devicetree.org/schemas/gpio/st,stmpe-gpio.yaml#
 $schema: http://devicetree.org/meta-schemas/core.yaml#
 
-title: STMicroelectonics Port Expander (STMPE) GPIO Block
+title: STMicroelectronics Port Expander (STMPE) GPIO Block
 
 description:
   STMicroelectronics Port Expander (STMPE) is a series of slow
